maquetado/scss/styles.scss

```diff
@@ -10,10 +10,29 @@
   width: 100%;
 }
 
+.book-cover-container {
+  display: block;
+  position: relative;
```

Comment:
If I'm not mistaken you can set this in `.book-cover` and it will work without having to use this container